apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -123,7 +123,7 @@ public function enqueue_assets() {
 
 		$default_info      = array(
 			'title'    => get_bloginfo( 'name' ),
-			'vertical' => $vertical_name,
+			'vertical' => $vertical['name'],
```

Comment:
Are we always confident this will be defined?

Reply:
It _should_ be. If it's not, our API responses need to be more consistent about status codes

Reply:
> should be

Still...never hurts to code defensively right? 😄

Reply:
I've added a guarding against an empty `$vertical`. Should I add one more guard for the name property? It's worth noting that frontend can handle an empty name by using a fallback.

Reply:
How about something like `$vertical['name'] ?: _x( 'Business', 'Default vertical name', 'full-site-editing' )`?

Reply:
We already have that very same logic on frontend to match how all other placeholders work. Do you want that duplicated? That feels weird

https://github.com/Automattic/wp-calypso/blob/873f5969f7cefd79e1032527eb315272c46611df/apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/utils/replace-placeholders.js#L6-L11

Reply:
Excellent point.